tests/cal_display_test.py

```diff
@@ -137,6 +137,23 @@ def test_str_week():
     '    20 21 22 23 24 25 26 ',
     '\x1b[1mM\xe4r \x1b[0m27 28 29  1  2  3  4 ']
 
+example_de_bsd = [
```

Comment:
I think a regex might be nicer here since there is very little difference in the outputs.